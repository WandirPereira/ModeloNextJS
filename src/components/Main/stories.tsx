import { Story, Meta } from '@storybook/react'
import Main from '.'

export default {
  title: 'Main',
  component: Main,
  args: {
    title: 'title default',
    description: 'description default'
  }
} as Meta

export const Basic: Story = (args) => <Main {...args} />
Basic.args = {
  title: 'React AvanÃ§ado',
  description: 'TypeScript, ReactJS, NextJS e Styled Components basic'
}

export const Default: Story = (args) => <Main {...args} />
