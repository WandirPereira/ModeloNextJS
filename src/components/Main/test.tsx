import { render, screen } from '@testing-library/react'

import Main from './'

describe('<Main />', () => {
  it('Should render the heading', () => {
    render(<Main />)
    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()
  })
})
