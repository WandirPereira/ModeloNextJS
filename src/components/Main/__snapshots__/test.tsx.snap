// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render the heading 1`] = `
<div>
  <h1>
    React Avan√ßado
  </h1>
</div>
`;
